examples/resources/vantage_folder/resource.tf

```diff
@@ -1,3 +1,10 @@
 resource "vantage_folder" "demo_folder" {
   title = "Demo Folder"
-}
+
+  # Include either the parent_folder_token or workspace_token
+  # If both are included, the API will use the parent_folder_token
+
+  # Uncomment one of the following:
+  workspace_token = "wrkspc_47c3254c790e9351"
+}# parent_folder_token = "fldr_47c3254c790e9351" 
```

Comment:
Think this should likely go above `workspace_token` after the comment, the formatting is odd here since it comes after the brace.

Reply:
how did that happen 😖